src/main/java/org/molgenis/utils/HgvsService.java

```diff
@@ -58,9 +58,9 @@ private boolean isEmptyValue(String value) {
   public String getHgvs(String nmTranscript, String cDNA, String ref, String alt, int start,
       int stop, String chromosome) {
     String hgvs;
-    // if (ref and alt not empty) or (transcript+cDNA would be invalid)
-    if (!(isEmptyValue(ref) && isEmptyValue(alt)) || (isEmptyValue(nmTranscript) || isEmptyValue(
-        cDNA) || nmTranscript.startsWith("NC"))) {
+    // if (ref and alt not empty) or (transcript+cDNA would be invalid) and chromosome != MT
+    if ((!(isEmptyValue(ref) && isEmptyValue(alt)) || (isEmptyValue(nmTranscript) || isEmptyValue(
+        cDNA) || nmTranscript.startsWith("NC"))) && !chromosome.equals("chrMT")) {
```

Comment:
Looks good here, in general I'd prefer `"chrMT".equals(chromosome)` instead of `chromosome.equals("chrMT")` cause the first does not explode if chromosome is null.